andela/healthchecks_turquoise
# [133846621] Deploy Project on Heroku Pipeline

#### What does this PR do?
Deploys this project onto a Heroku pipeline
#### Description of Task to be completed?
Configuration settings are added to the project:

-  gunicorn is configured to serve the project

-  static files are served using WhiteNoise

- Database migrations are run automatically before the application is deployed at any stage in the pipeline

Two Heroku applications are created, one for staging and the other for production:

- Staging at  ```healthchecks-turquoise-staging.herokuapp.com```
Upon tests (from CircleCI) passing on the develop branch, the project is deployed on this url

- Production at ```healthchecks-turquoise.herokuapp.com```
Upon test (from CircleCI) passing on the master branch, the project is deployed on this url

#### How should this be manually tested?
Upon approval of this PR, you should be able to test the application using the following two URLs:

- Go [healthchecks-turquoise-staging](https://healthchecks-turquoise-staging.herokuapp.com/) to  to view the project at staging.
- Go to [healthchecks-turquoise](https://healthchecks-turquoise.herokuapp.com) to view the project at production

#### What are the relevant pivotal tracker stories?
#134419027 - Setup Continuous Integration with CircleCI
